indy-besu/vdr/src/client/client.rs

```diff
@@ -30,27 +33,40 @@ impl LedgerClient {
     ///  client to use for building and sending transactions
     pub fn new(
         chain_id: u64,
-        node_address: &str,
+        mut node_addresses: Vec<&str>,
         contract_configs: &[ContractConfig],
     ) -> VdrResult<LedgerClient> {
+        let rpc_node_address = node_addresses.remove(0);
+
         trace!(
             "Started creating new LedgerClient. Chain id: {}, node address: {}",
             chain_id,
-            node_address
+            rpc_node_address
         );
 
-        let client = Web3Client::new(node_address)?;
+        let mut clients = node_addresses
+            .into_iter()
+            .map(|node_address| {
+                let client: Box<dyn Client> = Box::new(Web3Client::new(node_address)?);
+                Ok(Arc::new(client))
+            })
+            .collect::<Result<Vec<_>, VdrError>>()
+            .map_err(|_| VdrError::ClientNodeUnreachable)?;
+
+        let client = Box::new(Web3Client::new(rpc_node_address)?);
         let contracts = Self::init_contracts(&client, contract_configs)?;
 
+        clients.insert(0, Arc::new(client));
```

Comment:
I think RPC node should not participate in quorum check

Reply:
Sure. Removed